### DISCUSSION TITLE
Anfangserfahrung (kein Homekit-Button/Firewalleinstellungen)- Workaround - Frage zu Rolläden

### DISCUSSION THREAD

**Discussion by RoMueZo** [thkl/hap-homematic#416] [Q&A]
Hallo zusammen,
zunächst einmal vielen vielen Dank für das Programmieren dieses genialen Plugins!!

Nach jahrelanger Nutzung eines Mediola Gateways (V4 Plus) mit diversen Funkrollädenaktoren von Homematic und billigen Baumarkt Zwischensteckern mit 433Mhz hatte ich "Blut geleckt", da ich erfahren habe, dass man relativ einfach und kostengünstig auf einem RaspberryPi bzw. Raspberrymatic eine Möglichkeit schaffen kann, diese Geräte ins AppleHomekit einzubinden.
Jetzt bremst mich nur noch meine Frau, da ich viele "Visionen" von Fenstersensoren, Überwachungskameras, Wetterstationen usw. habe.. :)

Was das Thema Raspberrymatic anging, war/bin ich noch ziemlich unbedarft.
Nachdem ich mir eine solche zusammengebaut/gelötet hatte, wollte ich bilderbuchmäßig gemäß der Anleitung vorgehen. Mein Plan war, erst die die Raspberrymatic aufzusetzen, dann alle meine Rolläden einzubinden; dann das hap-homematic plugin zu installieren und fertig.
Es hört sich zwar blöd an, aber ich habe mich wirklich an die Anleitung gehalten (von allem die neusten Versionen) und auch brav alle Ports in der Firewall freigegeben, aber ich hab die Installation nicht hinbekommen. Irgendwie schien es so, als ob die Ports weiterhin blockiert waren, da sich das Plugin nicht installieren ließ und bei mir einfach kein HomeKit-Button auftauchte. Ich habe hier diverse Sachen ausprobiert; zig mal neugestartet; oft lange gewartet, ob sich was installiert usw.
Dann habe ich Diskussionen gelesen, dass man irgendwie in log-Dateien nachlesen solle, ob sich das Plugin überhaupt installieren würde.
Das mag zwar ein bisschen komisch und wahrscheinlich für jeden hier selbstverständlich klingen, aber wie komme ich denn überhaupt aus das "Dateisystem" der Raspberrymatic?
Naja, da ich das nicht kapiert hatte, hab ich mal einen Workaround ausprobiert:
Raspberrymatic neu aufgesetzt; dann direkt (mal kurz) alle Ports freigegeben und hap-homematic installiert und es lief!
Hmm, danach wieder die Ports geblockt und nur die benötigten freigegeben und es lief weiterhin.
Hätte ich mir mal die ganzen Seriennummern der Schaltaktoren aufgeschrieben, hätte ich nicht schon wieder durchs Haus laufen müssen, um diese anzulernen.. :)
Warum das so ist, kann ich nicht nachvollziehen, aber ich wollte meine Erfahrung gerne teilen, da vielleicht jemand anders auch die gleichen Probleme haben könnte.

Bei der Nutzung der Rolläden (HM-LC-BI1PBU-FM) ist mir jetzt nach den ersten Tage etwas aufgefallen, wo ich mir nicht sicher bin, ob das normal ist:
Wenn ich die "Home-App" auf dem IPhone starte, dauert es erst eine Weile, bis der aktuelle Status angezeigt wird; eine Weile steht dann da "reagiert nicht oder keine Rückmeldung" oder sowas. Wenn ich dann bis zu ner Minute gewartet habe, wird der Status korrekt angezeigt.
Schließe ich dann die App und starte diese neu, wird der Status direkt richtig angezeigt.
Dann ist etwas komisch: Wenn ich in der CCU z.B. die Rolläden auf 50 % runterfahre (was diese direkt erkennt und anzeigt), erkennt das "die Home App" aber erst, wenn diese geschlossen und neu gestartet wird.
Es wirkt auf mich so, als ob die Aktualisierungen in der CCU nicht immer direkt ans Homekit übertragen werden, sondern einen Verzug haben.
Das ist jetzt vielleicht eine blöde Frage, aber ist das normal?
Ich meine mich erinnern zu können, in irgendnem Forum zur Homebridge gelesen zu haben, dass langsame Reaktion auf Befehle über das Homekit irgendeine Einstellung bzw. eine Portfreigabe hier erforderlich waren; das finde ich aber leider nicht wieder..

**Comment by ChrWi** [comment]
Hallo @RoMueZo,
Was ist denn deine Homekit Zentrale? Das iPhone kann es nicht sein.

**Reply by RoMueZo** [reply]
Hallo!
Bis letzten Monat hatte ich immer ein aktuelles IPad eingesetzt; vor ein paar Wochen habe ich mir nunmehr einen HomepodMini gekauft.
Seitdem ist es bedeutend besser; es wirkt so, als hätte man das IPad immer erst "aufwecken" müssen, obwohl mir hier keine Stromsparmodi bekannt wären, welche ich eingestellt habe.
Sry, mein Post von April war da was missverständlich; ich habe die Geräte halt nur mit der HomeApp meines IPhones gesteuert..

**Reply by jb-home** [reply]
Das iPad geht von alleine in den Stromsparmodus. Da kann man nichts einstellen. Das mit dem 'Aufwecken' ist da schon der richtige Ansatz. Daher sind HomePods oder Apple TVs die 'besseren/schnelleren' HomeKit Zentralen.

**Comment by ChrWi** [comment]
Mit dem Wechsel auf einen HomePod Mini habe ich aktuell bei 14 Geräten (davon 8 BROLLs) und 6 Variablen in der Home App eine Aktualisierung von 5-7 Sekunden- sowohl im heimischen WLan als auch über LTE als auch aus dem Firmen-WLAN. Beim vorherigen iPad war das 15sek bis „gar nicht“.
